airflow/utils/dag_processing.py

```diff
@@ -715,11 +726,25 @@ def start(self):
                     self._add_callback_to_queue(agent_signal)
                 else:
                     raise AirflowException("Invalid message")
-            elif not self._async_mode:
+
+            simple_dags = []
+            for sentinel in ready:
+                processor = self.waitables.get(sentinel)
+                if not processor:
+                    continue
+
+                simple_dags = self._collect_results_from_processor(processor)
```

Comment:
```suggestion
                simple_dags = simple_dags + self._collect_results_from_processor(processor)
```

?

Reply:
actually scratch this. L730 can be removed

Reply:
Oh I thought I had removed L730. Yeah.